packages/frontend/src/components/global/MkAvatar.vue

```diff
@@ -110,32 +118,41 @@ watch(() => props.user.avatarBlurhash, () => {
 }
 
 .cat {
-	&:before, &:after {
-		background: #df548f;
-		border: solid 4px currentColor;
-		box-sizing: border-box;
-		content: '';
+	> .earLeft,
+	> .earRight {
+		contain: strict;
 		display: inline-block;
 		height: 50%;
 		width: 50%;
+		background: currentColor;
+
+		&::before {
+			contain: strict;
+			content: '';
+			display: block;
+			width: 75%;
+			height: 75%;
+			margin: 12.5%;
```

Comment:
```suggestion
			width: 60%;
			height: 60%;
			margin: 20%;
```

個人的にはこのくらいの方が好み

![image](https://user-images.githubusercontent.com/7973572/217854303-0895169a-fceb-4d20-a0f9-cb595e64f06c.png)

Reply:
それぐらいでもいいかも